tests/from.bats

```diff
@@ -26,12 +24,10 @@ load helpers
 
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json dir:${elsewhere})
   buildah rm $cid
-  buildah rmi ${elsewhere}
   [ "$cid" = elsewhere-img-working-container ]
 
   cid=$(buildah from --pull-always --signature-policy ${TESTSDIR}/policy.json dir:${elsewhere})
   buildah rm $cid
-  buildah rmi ${elsewhere}
   [ "$cid" = `basename ${elsewhere}`-working-container ]
 }
```

Comment:
Probably ought to add a 'rm -rf ${elsewhere}/* here at the end.   Might not hurt to add 'buildah rm --all' and 'buildah rmi --all' in that order too.

Reply:
Thanks. will do so.